src/main/java/org/redcraft/redcraftreboot/Reboot.java

```diff
@@ -0,0 +1,105 @@
+package org.redcraft.redcraftreboot;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.Bukkit;
+
+public class Reboot implements CommandExecutor {
+
+    private  final RedCraftReboot plugin;
+    TimeTask timeTask;
+
+    public Reboot(final RedCraftReboot plugin) {
+        this.plugin = plugin;
+
+    }
+
+    @Override
+    public boolean onCommand(final CommandSender commandSender, final Command command, final String s,
+            final String[] args) {
```

Comment:
Are you sure you don't need to check permissions? I don't remember if setting them in `plugin.yml` enforces them